tests/lib.rs

```diff
@@ -116,11 +110,11 @@ use std::path::PathBuf;
 #[case::es_2dcode_raw_2101_json("ES/2DCode/raw/2101.json")]
 #[case::es_2dcode_raw_2102_json("ES/2DCode/raw/2102.json")]
 #[case::es_2dcode_raw_2103_json("ES/2DCode/raw/2103.json")]
-#[ignore = "RSA signature. See #2"]
+#[ignore = "Key is P-384"]
```

Comment:
Should we open a dedicated issue for those?

Reply:
I agree: even if it's a problem related to the test files, at least we have a public issue (which could be also referenced from the official repos...)

Reply:
I created a pretty generic issue (#34). We could use it to track down all the strange issues with tests. @Rimpampa do you want to add `See #34.` in the ignored tests?